### PR TITLE
[image-rs] Added corporas to container.

### PR DESCRIPTION
**Pull request by kobrineli**

### CODE REVIEW

File: projects/image-rs/Dockerfile [hunk 1]
```diff
@@ -48,3 +48,19 @@ RUN ../build.sh
 RUN  scrpit=$(find /root/.cargo/registry/src/ -name update-libfuzzer.sh) && $scrpit 3c7960cba19ed926b8e86b4c619e81c0f7da4d15
 
 RUN ../build.sh
+
+# Default corpus.
+RUN mkdir /corpus
```
Comment (SweetVishnya) [inline projects/image-rs/Dockerfile:53]: We do not need empty corpus -- no input corpus is allowed for libFuzzer

File: projects/image-rs/guess.toml [hunk 2]
```diff
@@ -21,4 +21,4 @@ jobs = 2
 
 [libfuzzer]
 path = "/fuzzer_script_guess"
-args = "-rss_limit_mb=8192 -jobs=4 /fuzz/corpus"
+args = "-rss_limit_mb=8192 -jobs=4 /corpus"
```
Comment (SweetVishnya) [inline projects/image-rs/guess.toml:24]: just remove /corpus

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/image-rs/Dockerfile (hunk 1 above)
Comment (kobrineli) [reply projects/image-rs/Dockerfile]: done

File: projects/image-rs/guess.toml (hunk 2 above)
Comment (kobrineli) [reply projects/image-rs/guess.toml]: done

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**